examples/Advanced-Building-your-own-architecture.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>there is a typo in <code>continous.</code>  should be <code>continuous</code> . I do that typo often too :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Same :D thanks for the catch!</p>